content/rma_intro.tex

```diff
@@ -2,7 +2,7 @@
 mechanisms of the \openshmem{} \ac{API}. While using these mechanisms, the user
 is required to provide parameters only on the calling side. A characteristic of
 one-sided communication is that it decouples communication from the
-synchronization. One-sided communication mechanisms transfer the data but do not
+synchronization. One-sided communication mechanisms transfer the data, but do not
```

Comment:
I believe this comma is wrong. The second phrase is a dependent clause.  There seem to be other instances below that shouldn't be "fixed".

Reply:
I agree that this comma should be dropped. It is not joining two independent clauses with a coordinating conjunction.